Sources/Views/ALKChatCell.swift

```diff
@@ -205,6 +205,11 @@ public final class ALKChatCell: SwipeTableViewCell, Localizable {
         return view
     }()
 
+    private let customTagView: KMTagLabelsView = {
+        let v = KMTagLabelsView()
```

Comment:
Change var name to something meaningful.

Reply:
renamed to `tagLabelsView`